packages/core/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@viamrobotics/prime-core",
+  "version": "0.0.1",
+  "scripts": {
+    "dev": "vite dev",
+    "build": "vite build && npm run package",
+    "preview": "vite preview",
+    "package": "svelte-kit sync && svelte-package && publint",
+    "prepublishOnly": "npm run package",
+    "check": "concurrently -g pnpm:check-*",
+    "check-svelte": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
+    "check-lint": "prettier --check . && eslint .",
+    "format": "prettier --write .",
```

Comment:
Are we missing extensions on either of these calls? IIRC (could be wrong) using `.` will only check prettier and eslint defaults, which won't include `.svelte`, `.ts`, etc